pkg/controller/awsfederatedaccountaccess/awsfederatedaccountaccess_controller.go

```diff
@@ -309,25 +312,34 @@ func (r *ReconcileAWSFederatedAccountAccess) createIAMRole(awsClient awsclient.C
 	return nil
 }
 
-func (r *ReconcileAWSFederatedAccountAccess) createOrUpdateIAMPolicy(awsClient awsclient.Client, afr awsv1alpha1.AWSFederatedRole, afaa awsv1alpha1.AWSFederatedAccountAccess) error {
-
-	policy, err := r.createIAMPolicy(awsClient, afr, afaa)
+// TODO() this will need to be moved to a different package because it's shared across controller
+func CreateOrUpdateIAMPolicy(awsClient awsclient.Client, afr awsv1alpha1.AWSFederatedRole, policyARN string) error {
```

Comment:
This should be moved to utils to avoid importing this whole controller to the federated role controller